app/services/live_lookup/folio.rb

```diff
@@ -55,7 +55,7 @@ def status(item)
 
     def due_date(item)
       date = item.fetch('dueDate', nil)
-      Date.parse(date).strftime('%m/%d/%Y') if date
+      Time.zone.parse(date).strftime('%m/%d/%Y') if date
```

Comment:
The due date already has a time zone, right? I find that this better expresses the intention:
```suggestion
     DateTime.parse(date).in_time_zone.strftime('%m/%d/%Y') if date
```

Reply:
Or alternately:
```suggestion
      date.to_time.in_time_zone.strftime('%m/%d/%Y') if date
```